external/box2d/Box2D/wscript

```diff
@@ -6,95 +6,107 @@ from waflib import Build
 VERSION = '3.1.0'
 APPNAME = 'box2d'
 
+
 def configure(conf):
-    pass
+    print("Configuring Box2D...")
+    plat = conf.env.PLATFORM
+    if not plat:
+        conf.fatal('Pass --platform=...')
 
-def build(bld):
-    box2d = bld.stlib(features = 'c',
-                      includes = '.. include',
-                      source = bld.path.ant_glob('**/*.c'),
-                      target = 'box2d')
+    conf.env.SIMD_CFLAGS, conf.env.SIMD_CXXFLAGS = [], []
+    conf.env.NOSIMD_CFLAGS, conf.env.NOSIMD_CXXFLAGS = [], []
+
+    if plat in ('win32', 'x86_64-win32'):
+        conf.env.STLIB_ST = 'lib%s.lib'
+        conf.env.cstlib_PATTERN = 'lib%s.lib'
+        conf.env.cxxstlib_PATTERN = 'lib%s.lib'
+        conf.env.append_unique('CFLAGS', ['/std:c11'])
+
+    if plat == 'armv7-android':
+        conf.env.SIMD_CFLAGS += ['-mfpu=neon']
+        conf.env.SIMD_CXXFLAGS += ['-mfpu=neon']
 
-    name = "%s-%s" % (APPNAME, VERSION)
-    packagedir = '.'
+    if plat in ('wasm-web', 'wasm_pthread-web'):
+        simd_flags = ['-matomics', '-mbulk-memory', '-msimd128', '-msse2']
+        conf.env.SIMD_CFLAGS += simd_flags
+        conf.env.SIMD_CXXFLAGS += simd_flags
 
-    if bld.env.PLATFORM in ('win32', 'x86_64-win32'):
-        bld.env.STLIB_ST         = 'lib%s.lib'
-        bld.env.cstlib_PATTERN   = 'lib%s.lib'
-        bld.env.cxxstlib_PATTERN = 'lib%s.lib'
+    elif plat == 'js-web':
+        pass
 
-        box2d.env.append_unique('CFLAGS', '/std:c11') # for _Static_assert
+    conf.env.append_unique('CFLAGS', conf.env.SIMD_CFLAGS)
+    conf.env.append_unique('CXXFLAGS', conf.env.SIMD_CXXFLAGS)
+
+
+def build(bld):
+    srcs = bld.path.ant_glob('**/*.c')
+    incs = '.. include'
+
+    name = f"{APPNAME}-{VERSION}"
+    plat = bld.env.PLATFORM
+    install_dir = '${PREFIX}/%s/lib/%s/' % (name, plat)
+
+    if plat != 'js-web':
+        box2d = bld.stlib(features='c',
+                          includes=incs,
+                          source=srcs,
+                          target='box2d',
+                          cflags=bld.env.SIMD_CFLAGS,
+                          cxxflags=bld.env.SIMD_CXXFLAGS)
+        bld.install_files(install_dir, bld.env.cxxstlib_PATTERN % 'box2d')
+
+    box2d_nosimd = bld.stlib(features='c',
+                             includes=incs,
+                             source=srcs,
+                             target='box2d_nosimd',
+                             cflags=bld.env.NOSIMD_CFLAGS + ['-DBOX2D_DISABLE_SIMD'],
+                             cxxflags=bld.env.NOSIMD_CXXFLAGS + ['-DBOX2D_DISABLE_SIMD'])
+    bld.install_files(install_dir, bld.env.cxxstlib_PATTERN % 'box2d_nosimd')
+
+    if plat == 'js-web':
+        simd_name = bld.env.cxxstlib_PATTERN % 'box2d'
+        nosimd_name = bld.env.cxxstlib_PATTERN % 'box2d_nosimd'
+        nosimd_node = bld.path.get_bld().make_node(nosimd_name)
+        bld.install_as(install_dir + simd_name, nosimd_node)
```

Comment:
I don't understand this part. Is this to make a copy, to make sure we have two libs?
Not sure I like it, as it is confusing. But we'll go with it for now I think.